internal/collector/settings.go

```diff
@@ -19,6 +21,11 @@ import (
 	"go.opentelemetry.io/collector/otelcol"
 )
 
+const (
+	otelTemplatePath     = "otelcol.tmpl"
```

Comment:
just wondering if we should use go:embed  for this instead? 
If it is done this way, I think it would need to get added to the .nfpm config

Reply:
Yeah definitely need the embed, gonna add that after done with demoing.

Reply:
Done now ✅